admin/views/partial-settings-tab-video.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * @package WPSEO\Admin\Views
+ */
+
+if ( ! defined( 'WPSEO_VERSION' ) ) {
+	header( 'Status: 403 Forbidden' );
+	header( 'HTTP/1.1 403 Forbidden' );
+	exit();
+}
+
+if ( ! empty( $tab_video_url ) ) :
+
+	?>
+	<div class="wpseo-tab-video-container">
+		<div class="wpseo-tab-video-container__handle">
```

Comment:
This is a clickable `div`, but then it should be a `button`. This way you also don't need `e.preventDefault` in your JavaScript.

If you have trouble because of page submitting you can use `<button type="button">`. This requires more styling but is more accessible.